cibuildwheel/__main__.py

```diff
@@ -208,9 +216,21 @@ def build_in_directory(args: CommandLineArguments) -> None:
         architectures=options.globals.architectures,
     )
 
-    if args.print_build_identifiers:
+    if args.print_build_identifiers is not None:
```

Comment:
Actually, I think this is safer:

```suggestion
    if args.print_build_identifiers:
```

That would make it easier to pass this though in the action as an empty string.